index.html

```diff
@@ -0,0 +1,34 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" type="text/css" href="styles.css" />
+    <title>Portfolio setup and mobile first</title>
+</head>
+<body>
+    <div>
+        <nav class="navbar">
+            <a class="box 1" href="https://www.github.com/AlejandraKeber">Alejandra</a>
+            <a class="box 2" href="#Menu"><img src="./Images/Navbarmenu.png" alt="Menu"></a>
+        </nav>
+    </div>
+    <div>
+        <h1 class="headline">I'm Alejandra! Glad to see you!</h1>
+    </div>
+    <div>
+        <p class="headlinecont">I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
+    </div>
+    <div>
```

Comment:
Kindly wrap this section inside a `section` tag , this will make your code more professional 👍 😄